seyik/app.js

```diff
@@ -25,17 +25,50 @@ appDataSource
     appDataSource.destroy();
   });
 
-  
 const app = express();
+const PORT = process.env.PORT;
 
 app.use(express.json());
 app.use(cors());
 app.use(morgan("dev"));
 
-app.get("/ping", function (req, res, next) {
+app.get("/ping", function (req, res) {
   res.json({ message: "pong" });
 });
```

Comment:
```suggestion
app.get("/ping", function (req, res) {
  return res.status(200).json({ message: "pong" });
});
```
단순한 용도의 health check api라고 하더라도, 백엔드에서 적절한 status code를 핸들링하여 반환할 수 있도록 수정해주세요.